profile.md

```diff
@@ -0,0 +1,3 @@
+# This is my profile 
+- My name
+- My job
```

Comment:
Please add email account to your profile

Reply:
Done my Boss!